library/core/src/macros/panic.md

```diff
@@ -24,11 +24,9 @@ The multi-argument form of this macro panics with a string and has the
 
 See also the macro [`compile_error!`], for raising errors during compilation.
 
+[ounwrap]: ../std/option/enum.Option.html#method.unwrap
```

Comment:
```suggestion
[ounwrap]: Option::unwrap
```

Reply:
Do links from core to std work now?

Reply:
`Option` is in core, not std: https://doc.rust-lang.org/core/option/enum.Option.html. The existing link is a re-export of core.

Reply:
Ah, forgot that std re-exports it :)